website_sale_cart_preview/static/src/js/website_sale_cart_preview.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
```

Comment:
using multiple $(document).ready() (Odoo already uses it in several places) it is not always the neatest thing to do.
maybe you can use animationRegistry to add your JS code limited only to the cart DOM selector, an example here https://github.com/odoo/odoo/blob/8.0/addons/mass_mailing/static/src/js/website_mass_mailing.js#L6